examples/convert_ieeg_to_bids.py

```diff
@@ -0,0 +1,167 @@
+# Generally, :func:`write_raw_bids` tries to extract as much
+# meta data as possible from the raw data and then formats it in a BIDS
+# compatible way. :func:`write_raw_bids` takes a bunch of inputs, most of
+# which are however optional. The required inputs are:
+#
+# * :code:`raw`
+# * :code:`bids_basename`
+# * :code:`bids_root`
+#
+# ... as you can see in the docstring:
+print(write_raw_bids.__doc__)
+
+###############################################################################
+# Let us initialize some of the necessary data for the subject
+# There is a subject, and specific task for the dataset
+subject_id = '001'  # zero padding to account for >100 subjects in this dataset
+task = 'testresteyes'
+bids_root = os.path.join(mne_dir, 'eegmmidb_bids')
+
+###############################################################################
+# Now we just need to specify a few more EEG details to get something sensible:
+# Brief description of the event markers present in the data. This will become
+# the `trial_type` column in our BIDS `events.tsv`. We know about the event
+# meaning from the documentation on PhysioBank
+
+# Now convert our data to be in a new BIDS dataset.
+bids_basename = make_bids_basename(subject=subject_id,
+                                   task=task,
+                                   acquisition="ecog")
+
+# need to set the filenames if we are initializing data from array
+with tempfile.TemporaryDirectory() as tmp_root:
+    tmp_fpath = os.path.join(tmp_root, "test_raw.fif")
+    raw.save(tmp_fpath)
+    raw = mne.io.read_raw_fif(tmp_fpath)
+
+    write_raw_bids(raw, bids_basename,
+                   bids_root,
+                   overwrite=True)
+
+###############################################################################
+# Step 3: Check and compare with standard
+# ---------------------------------------
+# Now we have written our BIDS directory.
+print_dir_tree(bids_root)
+
+###############################################################################
+# MNE-BIDS has created a suitable directory structure for us, and among other
+# meta data files, it started an `events.tsv` and `channels.tsv` and made an
+# initial `dataset_description` on top!
+#
+# Now it's time to manually check the BIDS directory and the meta files to add
+# all the information that MNE-BIDS could not infer. For instance, you must
+# describe EEGReference and EEGGround yourself. It's easy to find these by
+# searching for "n/a" in the sidecar files.
+#
+# Remember that there is a convenient javascript tool to validate all your BIDS
+# directories called the "BIDS-validator", available as a web version and a
+# command line tool:
+#
+# Web version: https://bids-standard.github.io/bids-validator/
+#
+# Command line tool: https://www.npmjs.com/package/bids-validator
+
+###############################################################################
+# Step 4: Plot output channels and check that they match!
```

Comment:
I don't understand the purpose of the second subplot here

Reply:
Got rid of that second subplot. I figured sometimes ppl like to see... 3d? heh.